### PR TITLE
Feature/vijay change wallpaper

### PR DESCRIPTION
**Pull request by mob-dev**
- Create Change Wallpaper page & Wallpaper preview page
- Save changed wallpaper in SharedPreferences

┆Issue is synchronized with this [Jira Task](https://highvibe.atlassian.net/browse/HVN1-209) by [Unito](https://www.unito.io/learn-more)

### CODE REVIEW

File: mobile/lib/modules/change_theme/widgets/wallpaper.dart [hunk 1]
```diff
@@ -32,67 +34,84 @@ class _WallpaperListState extends State<WallpaperList> {
...
-          child: InkWell(
-            onTap: () {
-              print('apply clicked..!!');
+          child: GradientRaisedButton(
+            label: Strings.apply,
+            onPressed: () {
+              if (selectedThemeIndex != -1) {
+                homeController.changeTheme(selectedThemeIndex);
+                Modular.to.pop();
+                showToast(message: 'Wallpaper changed successfully!!!');
```
Comment (AlexVegner) [inline mobile/lib/modules/change_theme/widgets/wallpaper.dart:52]: Tost overlap button, looks not good. 
Romove it.

File: mobile/lib/modules/change_theme/widgets/wallpaper.dart [hunk 2]
```diff
@@ -32,67 +34,84 @@ class _WallpaperListState extends State<WallpaperList> {
...
+              } else {
+                showToast(message: 'Please select wallpaper');
```
Comment (AlexVegner) [inline mobile/lib/modules/change_theme/widgets/wallpaper.dart:54]: User should not have option to not select one of background wallpaper
Remove showToast

File: mobile/lib/modules/change_theme/widgets/wallpaper.dart [hunk 3]
```diff
@@ -1,29 +1,31 @@
 import 'package:core/common/assets/assets.dart';
-import 'package:core/common/themes/themes.dart';
 import 'package:core/common/values/strings.dart';
 import 'package:flutter/material.dart';
+import 'package:flutter_modular/flutter_modular.dart';
```
Comment (AlexVegner) [inline mobile/lib/modules/change_theme/widgets/wallpaper.dart:4]: Wrong padding 
Screenshot 2020-08-31 at 17 44 12

File: mobile/lib/modules/change_theme/widgets/wallpaper.dart [hunk 4]
```diff
@@ -1,29 +1,31 @@
...
+  WallpaperList({this.selectedThemeIndex});
 
   @override
   _WallpaperListState createState() => _WallpaperListState();
 }
 
 class _WallpaperListState extends State<WallpaperList> {
   int selectedThemeIndex = -1;
+  final homeController = Modular.get<HomeController>();
+  final sharedPreferences = Modular.get<SharedPreferences>();
```
[the remark this answers is not in the document]
Comment (mob-dev) [reply mobile/lib/modules/change_theme/widgets/wallpaper.dart:23]: Done

File: mobile/pubspec.yaml [hunk 5]
```diff
@@ -51,6 +51,9 @@ dependencies:
   video_player: ">=0.10.11+1 <2.0.0"
   virtual_keyboard: ^0.1.4
   web_socket_channel: ^1.1.0
+  web3dart: ^1.2.3
```
Comment (AlexVegner) [inline mobile/pubspec.yaml:54]: Already presented in core, avoid to add same dependency to mobile or web if it presented in core
```
web3dart: ^1.2.3
google_sign_in: ^4.4.6
```

**Review by AlexVegner — CHANGES_REQUESTED** [review]

**Review by AlexVegner — APPROVED** [review]

**Pull request merged**